platformcomponents/desktop/attachment-file-error.json

```diff
@@ -0,0 +1,22 @@
+{
+  "attachment-file-error": {
+    "sent": {
+      "comment": "Applied to sent & blocked file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5533%3A11688",
+      "background": {
+        "#normal": "@theme-background-secondary-normal",
+        "#hovered": "@theme-background-secondary-hover",
+        "#active": "@theme-background-secondary-active"
+      },
+      "border": "@theme-outline-secondary-normal",
+      "previewBox": {
+        "icon": "@theme-common-text-white",
+        "background": "@theme-background-accent-normal"
+      },
+      "label": {
+        "primary": "@theme-text-primary-disabled",
+        "secondary": "@theme-text-primary-disabled"
+      }
```

Comment:
This component also has non-disabled text states, like [here](https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5533%3A11688). I see text-primary-normal, text-secondary-normal...

Reply:
Pushed commit
[Added missing primary and secondary label states for attachment-file-error](https://github.com/momentum-design/tokens/pull/174/commits/2f7585787a11a523658efa7d5ad98fb1f942ea1c)
Changing this conversation to Resolved.